Specs/DateTime/French/DateExtractor.json

```diff
@@ -947,7 +947,7 @@
   },
   {
     "Input": "Qui ai-je envoye un mois il y a",
-    "NotSupported": "javascript, dotnet, python",
+    "NotSupported": "javascript, dotnet, python, java",
```

Comment:
I assumed that since this was not supported by any of the other languages, it would not be supported for Java either.

Reply:
Yes, it should be OK to temporarily mark it as NotSupported until the rest of the support is ported.